### ISSUE TITLE
Re-order the Theme list, from Darker to Light to make them easier to spot lighter and darker themes

### ISSUE THREAD

**Issue by DarrenTAnims** [tahoma2d/tahoma2d#557]
**Describe the bug**
It's not exactly a bug, to be fair, but it would make choosing a Tahoma easier. And I hope it should be a quick thing to fix.
The list of themes in the Themes dropdown in the Interface section of the Preferences. The list would be better if it was:

- Darker
- Dark
- Medium
- Neutral
- Light

**Screenshots**
![image](https://user-images.githubusercontent.com/28575907/104237892-a78bb580-5450-11eb-92d9-c3d726143113.png)

**Comment by jeremybullock** [comment]
I agree, but the list is created alphabetically.  This is partially based around the freedom for users to create their own themes and have them show up in the list.
I would be fine with not allowing user defined themes and only using what's in the list.  Then we could put them in an order that makes more sense.
